dev-utils/karma.js

```diff
@@ -43,7 +43,7 @@ const { tunnelIdentifier } = getSauceConnectOptions()
  * Common base config for all the mono repo packages
  */
 const baseConfig = {
-  files: [specPattern],
+  files: [require.resolve('babel-polyfill/dist/polyfill.js'), specPattern],
```

Comment:
Any reason why we need to polyfill all shims instead of promise polyfill alone?.

Reply:
This is needed for the async functions in the tests.